global.json

```diff
@@ -10,6 +10,6 @@
     "xcopy-msbuild": "17.7.4"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.23463.1"
+    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.23508.1"
```

Comment:
This is not ok, we need to stay on Arcade 8

Reply:
Does the major arcade update generly need to go only in hand with sdk update?

Reply:
The most relevant factor is support lifecycle and what .NET versions we're going to ship in. We will be shipping in .NET 8 SDKs (as well as .NET 9-preview SDKs) until .NET 9, so we must not move forward past the .NET 8 toolchain until the release where we won't ship into .NET 8 any more because it's .NET SDK 9.0.100.

Reply:
Gotcha.
Noted.